fixes.inc

```diff
@@ -2233,15 +2249,51 @@ static stock
 	 * A record of which menus have and haven't been shown yet.
 	 */
 	FIXES_gsValidMenus[_FIXES_CEILDIV(MAX_MENUS, cellbits)],
+#if FIXES_SilentKick && (FIX_OnVehicleMod || FIX_OnPlayerEnterVehicle || FIX_OnPlayerEnterVehicle_2)
 	/*
 	 * FIXES_gsPlayerIP[MAX_PLAYERS]
 	 *
 	 * A player's IP as a 32-bit integer.
 	 */
-#if FIXES_SilentKick && (FIX_OnVehicleMod || FIX_OnPlayerEnterVehicle || FIX_OnPlayerEnterVehicle_2)
```

Comment:
Now I see better, this have no effect at all so I replaced it to its original position.
https://github.com/Open-GTO/sa-mp-fixes/pull/64/commits/9c113e0d4dec89173612e3ce353a7f239f01c6b6